pkg/packagekit/autoupdates.jsx

```diff
@@ -320,41 +380,54 @@ export default class AutoUpdates extends React.Component {
                             <Select.SelectEntry data="sat">{_("Saturdays")}</Select.SelectEntry>
                             <Select.SelectEntry data="sun">{_("Sundays")}</Select.SelectEntry>
                         </Select.Select>
-                    </span>
 
-                    <span className="auto-conf-group">
                         <span className="auto-conf-text">{_("at")}</span>
 
-                        <Select.Select id="auto-update-time" enabled={enabled} initial={backend.time}
-                                       onChange={ t => this.handleChange(null, null, null, t) }>
-                            { hours.map(h => <Select.SelectEntry key={h} data={h + ":00"}>{('0' + h).slice(-2) + ":00"}</Select.SelectEntry>)}
+                        <Select.Select id="auto-update-time" enabled={enabled} initial={this.state.time}
+                                       onChange={t => this.setState({ time: t })}>
+                            {hours.map(h => <Select.SelectEntry key={h} data={h + ":00"}>{('0' + h).slice(-2) + ":00"}</Select.SelectEntry>)}
                         </Select.Select>
-                    </span>
-
-                    <span className="auto-conf-group auto-conf-text">{_("and restart the machine automatically.")}</span>
-                </div>
-            );
-        }
-
-        // we want the button to already show the target state while being disabled
-        const onOffState = this.state.pendingEnable == null ? backend.enabled : this.state.pendingEnable;
-
-        return (
-            <div className="header-buttons pk-updates--header pk-updates--header--auto" id="automatic">
-                <h2 className="pk-updates--header--heading">{_("Automatic updates")}</h2>
-                <div className="pk-updates--header--actions">
-                    <OnOffSwitch state={onOffState} disabled={!enabled}
-                                 onChange={e => {
-                                     if (!this.state.backend.installed) {
-                                         install_dialog(this.state.backend.packageName)
-                                                 .then(() => { this.initializeBackend(true).then(() => { this.handleChange(e, null, null, null) }) }, () => null);
-                                     } else {
-                                         this.handleChange(e, null, null, null);
-                                     }
-                                 }} />
-                </div>
-                {autoConfig}
-            </div>);
+                    </FormGroup>
+
+                    <Alert variant="info" title={_("This host will reboot after updates.")} isInline />
+                </>}
+            </Form>
+        );
+
+        return (<>
+            <Button variant="primary"
```

Comment:
PF4 documentation suggests that there should only be one primary button per page, I would make this secondary.

Reply:
Fixed